docs/user/intro.md

```diff
@@ -6,51 +6,36 @@ sidebar_position: 0
 
 # Welcome
 
-Solus is committed to helping our users get the most out of their systems. The Help Center has been put together to introduce Solus users to our ecosystem and guide them through some of the most common tasks they might need help with.
+Solus is committed to helping users get the most out of their systems. We designed the help center to introduce our ecosystem and guide you through typical tasks you might need help with.
 
-## Sections
+## Table of contents
 
-With that in mind, the Help Center has been broken up into several sections to help better organize the docs within:
+We organized the help center into sections to help you navigate the contents easily:
 
-**[Quick Start](/docs/category/quick-start)**
+| Section                                              | Description                                                                                                                                                                                 |
+| ---------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Quick Start](/docs/category/quick-start)            | If you're new to Solus, this section will set you on the right track. Find documentation on installing Solus, software management, the boot process, and troubleshooting.                   |
+| [Contributing](/docs/category/contributing)          | Find how you can contribute to Solus, whether you're a beginner or an experienced Linux and Solus user.                                                                                     |
+| [Editions](/docs/user/editions/)                     | Understand the different editions of Solus: differences between editions, configuration tasks, and helpful tips and tricks.                                                                |
+| [Hardware](/docs/user/hardware/)                     | Solus supports most hardware out-of-the-box, but there are exceptions. This section focuses on those exceptions and how to get the most out of your hardware. |
+| [Package management](/docs/user/package-management/) | Learn how to install software and manage the repositories of your Solus system.                                                                                                             |
```

Comment:
This gives me a 404 when I test locally. I think the trailing slash may need to be removed